wcf.targets

```diff
@@ -47,4 +47,9 @@
      <WcfToolsOutputPath>$(WcfRootFolder)bin\Wcf\tools\</WcfToolsOutputPath>
   </PropertyGroup>
 
+  <!-- Override ValidateIgnoreReference from Target 'ValidatePackageTargetFramework' in PackageLibs.targets -->
+  <ItemGroup>
+    <ValidateIgnoreReference Include="System.Private.ServiceModel" />
+  </ItemGroup>
+
```

Comment:
@ericstj it took me a little while to discover this in PackageLibs.targets to get passed the indirect reference failures. I noticed that System.Private.Uri was added to this collection in the coreclr repo, I believe for Wcf we can just override it safely in our own repo.

It all builds and the packages look correct, could you confirm?